install_ant-media-server.sh

```diff
@@ -99,6 +99,10 @@ check $?
 $SUDO mkdir $AMS_BASE/log
 check $?
 
+$SUDO cp $AMS_BASE/antmedia.service /lib/systemd/system/
```

Comment:
Do we need below lines?

`$SUDO cp $AMS_BASE/antmedia /etc/init.d/
check $?
$SUDO update-rc.d antmedia defaults
check $?
$SUDO update-rc.d antmedia enable
check $?
`